pages/rfc/registry-updates.rst

```diff
@@ -27,14 +27,91 @@ DRAFT
 Terminology
 ***********
 
+TODO
+
 - Event: an event in the registry event log, e.g. the RegisterVendor event
 - Event payload: the actual contents of the event, describing an object.
 - Object: the thing an event describes; either a vendor, organization or endpoint.
 - Event header: top-level properties of an event describing its payload.
 
-Challenges
+Operations
 **********
 
+The mutate operations that can be performed (by human operators or systems) on the registry are as follows:
+
+**Vendors**
+
+* Register vendor (currently supported)
+* Update vendor name
+* Reissue CA certificate
+* Rotate keys (and reissue associated certificates)
+* Deactivate/reactivate vendor
+
+**Organizations**
+
+* Register (claim) organization as vendor (currently supported)
+* Update organization name
+* Reissue CA certificate
+* Rotate keys (and reissue associated certificates)
+* Migrate organization to another vendor
+
+**Endpoints**
+
+* Register endpoint for organization (currently supported)
+* Update endpoint (URL, type, properties)
+* Deactivate/reactivate endpoint
+
+.. note::
+
+    There are other operations that are somewhat related to the registry like issuing a TLS certificate to another vendor
+    or revoking a certificate, but these don't mutate the registry and thus aren't relevant for this RFC.
```

Comment:
What would then the state be for a revoked certificate? Still in the registry but not able to connect to? Do we need an internal state? Revoked + in registry == deactivated?